src/data/mystery-encounters/mystery-encounter-utils.ts

```diff
@@ -342,7 +397,10 @@ export async function initBattleWithEnemyConfig(scene: BattleScene, partyConfig:
       enemyPokemon.setX(-66 + enemyPokemon.getFieldPositionOffset()[0]);
       enemyPokemon.resetSummonData();
     }
-
+    // we can't use trysetstatus because pokemon aint in battle yet
+    if (partyConfig?.status) {
```

Comment:
Status field should be moved to the enemy pokemon config level rather than the party config level I think. Otherwise a party config with multiple pokemon will force all pokemon to have the same status

Reply:
You can then move the status application logic from here to the loop just below that sets all the other custom params for enemy pokemon.